client/src/components/Explore.js

```diff
@@ -42,57 +47,55 @@ function Explore() {
 	];
 
 	return (
-		<Grid
-			container
-			justify="center"
-			alignItems="center"
-			direction="row"
-			className={classes.pageContainer}
-		>
+		<>
 			<Grid
 				container
-				item
-				xs={12}
 				justify="center"
 				alignItems="center"
-				className={classes.pt4}
+				direction="row"
+				className={classes.pageContainer}
 			>
 				<Grid
+					container
 					item
 					xs={12}
+					justify="center"
+					alignItems="center"
+					className={classes.pt4}
+				>
+					<Grid item xs={12} container justify="center" alignItems="center">
+						<Typography variant="h2" className={classes.textAlign}>
+							Explore Destinations
+						</Typography>
+					</Grid>
+				</Grid>
+				<Grid item>
+					<Typography variant="h5" className={classes.headerSpacing}>
+						World's Top Destinations to Explore
+					</Typography>
+				</Grid>
+				<Grid
 					container
+					item
 					justify="center"
 					alignItems="center"
+					spacing={5}
+					className={classes.cardContainer}
 				>
-					<Typography variant="h2">Explore Destinations</Typography>
+					{locations.map((loc) => {
+						return (
+							<Grid item xs={12} lg={3}>
```

Comment:
You could return the <Grid item> from within the ExploreCard component so the Explore Card would return <Grid><Paper>...</Paper></Grid>. That way, the map function would just return one line of code. Up to you, just thought it might look more modular.